art/estimators/certification/derandomized_smoothing/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+Randomized smoothing estimators.
```

Comment:
Is it DeRandomized or Randomized?
```suggestion
DeRandomized smoothing estimators.
```